etc/kayobe/ansible/requirements.yml

```diff
@@ -11,6 +11,16 @@ roles:
   - name: ansible-lockdown.rhel8_cis
     src: https://github.com/ansible-lockdown/RHEL8-CIS
     version: 1.3.0
+  - name: ansible-lockdown.ubuntu22_cis
+    src: https://github.com/stackhpc/UBUNTU22-CIS
+    #FIXME: Waiting for https://github.com/ansible-lockdown/UBUNTU22-CIS/pull/132
+    # to be in a tagged release
+    version: feature/inject_facts_as_vars
+  - name: ansible-lockdown.rhel9_cis
+    src: https://github.com/ansible-lockdown/RHEL9-CIS
+    #FIXME: Waiting for https://github.com/ansible-lockdown/RHEL9-CIS/pull/54
```

Comment:
This PR is now closed, but I think the content has been moved and merged

Reply:
Yes, it has been merged.

Reply:
Checked and still only in devel